### ISSUE TITLE
[Bug]: Wrong derivation path of Ledger Live wallet

### ISSUE THREAD

**Issue by tuler** [blocknative/web3-onboard#880]
### Current Behavior

When selecting "Ledger Live" as the derivation path of the Ledger wallet it is listing accounts using the following derivation paths:

```
m/44'/60'/0
m/44'/60'/1
m/44'/60'/2
m/44'/60'/3
(...)
```

### Expected Behavior

It should be listing accounts with the following paths instead:

```
m/44'/60'/0'/0/0
m/44'/60'/0'/0/1
m/44'/60'/0'/0/2
m/44'/60'/0'/0/3
(...)
```

### Onboard Version

2.0.9

### Node Version

16

### What browsers are you seeing the problem on?

Chrome

**Comment by aaronbarnardsound** [comment]
I can replicate this issue, will have a fix out asap!

**Comment by aaronbarnardsound** [comment]
Closed via #901
